Dockerfile

```diff
@@ -1,17 +1,23 @@
-
 FROM ruby:2.4.1
 MAINTAINER Conor Heine <conor.heine@gmail.com>
```

Comment:
MAINTAINER is just additional metadata used to identify the owner of the image. It doesn't affect behavior, nor is it used by Docker when the container is run.

Use of MAINTAINER is sort of deprecated though. It's recommended practice to use LABEL with MAINTAINER. i.e:

    LABEL MAINTAINER="Connor Heine <connor.heine@gmail.com>"

It doesn't really matter in this specific case because the images don't have any other labels, but the above method is preferred because multiple labels can be added to an image in a single layer whereas the dedicated MAINTAINER will cause two layers to be created.